src/Tickets/Emails/Email/RSVP.php

```diff
@@ -0,0 +1,319 @@
+		$heading = apply_filters( "tec_tickets_emails_{$this->slug}_heading_plural", $heading, $this->id, $this->template );
+
+		return $this->format_string( $heading );
+	}
+
+	/**
+	 * Get default email subject.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_subject(): string {
+		return sprintf(
+			// Translators: %s - Lowercase singular of rsvp.
+			esc_html__( 'Your %s from {site_title}', 'event-tickets' ),
+			tribe_get_rsvp_label_singular()
+		);
+	}
+
+	/**
+	 * Get default email subject for plural rsvps.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_subject_plural() {
+		return sprintf(
+			// Translators: %s - Lowercase plural of rsvps.
+			esc_html__( 'Your %s from {site_title}', 'event-tickets' ),
+			tribe_get_rsvp_label_plural()
+		);
+	}
+
+	/**
+	 * Get subject for plural rsvps.
+	 * 
+	 * @since TBD
+	 * 
+	 * @return string
+	 */
+	public function get_subject_plural(): string {
+		$option_key = $this->get_option_key( 'subject-plural' );
+		$subject = tribe_get_option( $option_key, $this->get_default_subject_plural() );
+
+		// @todo: Probably we want more data parsed, or maybe move the filters somewhere else as we're always gonna
+
+		/**
+		 * Allow filtering the email subject globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $subject  The email subject.
+		 * @param string $id       The email id.
+		 * @param string $template Template name.
+		 */
+		$subject = apply_filters( 'tec_tickets_emails_subject_plural', $subject, $this->id, $this->template );
+
+		/**
+		 * Allow filtering the email subject.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $subject  The email subject.
+		 * @param string $id       The email id.
+		 * @param string $template Template name.
+		 */
+		$subject = apply_filters( "tec_tickets_emails_{$this->slug}_subject_plural", $subject, $this->id, $this->template );
+
+		return $this->format_string( $subject );
+	}
+
+	/**
+	 * Get email settings fields.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_settings_fields(): array {
+		return [
+			[
+				'type' => 'html',
+				'html' => '<div class="tribe-settings-form-wrap">',
+			],
+			[
+				'type' => 'html',
+				'html' => '<h2>' . esc_html__( 'RSVP Email Settings', 'event-tickets' ) . '</h2>',
+			],
+			[
+				'type' => 'html',
+				'html' => '<p>' . esc_html__( 'Registrants will receive an email including their RSVP info upon registration. Customize the content of this specific email using the tools below. The brackets {event_name}, {event_date}, and {rsvp_name} can be used to pull dynamic content from the RSVP into your email. Learn more about customizing email templates in our Knowledgebase.' ) . '</p>',
+			],
+			$this->get_option_key( 'enabled' ) => [
+				'type'                => 'toggle',
+				'label'               => esc_html__( 'Enabled', 'event-tickets' ),
+				'default'             => true,
+				'validation_type'     => 'boolean',
+			],
```

Comment:
We're missing the `Use Ticket Email` described on the Jira ticket.

Reply:
Well... I noticed that the JIRA ticket referenced a lot of "ticket" info, so I assumed it was copied over and not yet updated. I'll add it, though.

Reply:
The idea is that, as a user, you can fall back to the Ticket email template and settings in case you want to have the same thing for both.

Reply:
The other things referencing tickets (subject, heading, and so on) were also intentional. Missed that on the CR.

Reply:
Oh...

Reply:
Added it. I also added the logic to not show the remaining settings if they opt to use the ticket email settings instead.